shared/ff_util/tools/config_tool.cc

```diff
@@ -0,0 +1,90 @@
+/* Copyright (c) 2017, United States Government, as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ *
+ * All rights reserved.
+ *
+ * The Astrobee platform is licensed under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+// Command line flags
+#include <gflags/gflags.h>
+#include <gflags/gflags_completions.h>
+
+// Include RPOS
+#include <ros/ros.h>
+
+// FSW includes
+#include <ff_util/ff_names.h>
+#include <ff_util/ff_flight.h>
+#include <ff_util/ff_action.h>
+#include <ff_util/ff_serialization.h>
+#include <ff_util/config_client.h>
+
+// C++ STL includes
+#include <iostream>
+#include <sstream>
+#include <fstream>
+#include <string>
+#include <memory>
+
+// Gflags
+DEFINE_string(ns, "", "Robot namespace");
+DEFINE_string(node, "", "Localization pipeline (no, ml, ar, hr)");
+DEFINE_string(parameter, "nominal", "Flight mode");
+DEFINE_string(type, "", "Path planning algorithm");
```

Comment:
I don't think the help messages here are right

Reply:
I actually found out that there is already a reconfigure tool!

Reply:
Even better!